PR for configuration with 2 strings and 3 phases

maubreville opened the pull request:
First of all: Thanks for this magnificent library! It works very nicely, after a little change I implemented. Apparently the configuration here is 3 phases but 2 strings, which was not yet covered by the current version of the library.

maubreville committed:
Added option for 2 strings with 3 phases

In the PV inverter here I found an installation with three phases and 2 strings, which was not covered up until now for this library.

maubreville committed:
Update kostalpiko.py

Added option for 2 strings / 3 phases, which is apparently also possible :-) 

Thanks for this great library!

maubreville committed:
Update / fix for last commit

Forgot to import the new constant DOUBLE_STRING_THREE_PHASES_INDICES

rcasula commented on kostalpiko/const.py:
```diff
@@ -22,6 +22,17 @@
+    "l2_voltage": 8,
+    "string2_current": 9,
+    "l2_power": 10,
+    "l3_voltage": 8,
```
why `l3_voltage ` and `l2_voltage` have the same index?

rcasula commented on kostalpiko/const.py:
```diff
@@ -22,6 +22,17 @@
+    "string2_current": 9,
+    "l2_power": 10,
+    "l3_voltage": 8,
+    "l3_power": 10,
```
same here with `l2_power`

maubreville committed:
Update const.py

Shouldn't have done a PR close to midnight ;-) Thanks for catching the bug with the indices.

maubreville commented:
Thanks! 👍

rcasula commented:
Thank you!
